en_US/getting-started/install-ee.md

```diff
@@ -138,9 +138,20 @@ ZIP packages are released for quick testing and hot-beam upgrade. Do NOT install
 
 ## Install EMQX in Docker (Contain a simple docker-compose cluster)
 
-:::
-Because emqx stores data in the 'data/mnesia/<node name>' directory, when using the container to start emqx,
-the hostname or FQDN must be used to form the node name of emqx. Otherwise, the data storage directory will be switched, resulting in data loss.
+:::tip Note
+
+1. If you want to persist the EMQX Docker container, you need to keep the following directories, so that the data will persist even if the container no longer exists.
```

Comment:
```suggestion
1. If you want to persist data generated in the EMQX Docker container, you need to keep the following directories, so that the data will persist even if the container no longer exists.
```